### PR TITLE
fix(icons): dashboard icon switch

### PR DESCRIPTION
**Pull request by doralchan**
- Updated Dashboard icon and swapped it out in the sidebar
- Minor, minor tweak to switch icon to make it shorter

**Before & After:**
Screen Shot 2022-02-23 at 12 07 43 PM

**Before & After:**
Screen Shot 2022-02-23 at 12 13 04 PM
Screen Shot 2022-02-23 at 12 13 09 PM

### CODE REVIEW

**Review by evanpurkhiser — APPROVED** [review]
I like that

File: static/app/icons/iconSwitch.tsx [hunk 1]
```diff
@@ -5,10 +5,10 @@ import SvgIcon, {SVGIconProps} from './svgIcon';
 const IconSwitch = React.forwardRef<SVGSVGElement, SVGIconProps>((props, ref) => {
   return (
     <SvgIcon {...props} ref={ref}>
-      <path d="M3.91001 15.49C3.81147 15.4905 3.71383 15.4712 3.62281 15.4335C3.5318 15.3957 3.44925 15.3401 3.38001 15.27L0.510008 12.39C0.439251 12.3211 0.383015 12.2386 0.344618 12.1476C0.30622 12.0566 0.286438 11.9588 0.286438 11.86C0.286438 11.7612 0.30622 11.6634 0.344618 11.5724C0.383015 11.4814 0.439251 11.3989 0.510008 11.33L3.38001 8.46C3.52218 8.32752 3.71023 8.25539 3.90453 8.25882C4.09883 8.26225 4.28422 8.34096 4.42163 8.47837C4.55904 8.61579 4.63775 8.80117 4.64118 8.99547C4.64461 9.18978 4.57249 9.37782 4.44001 9.52L2.10001 11.86L4.44001 14.21C4.51077 14.2789 4.567 14.3614 4.6054 14.4524C4.6438 14.5434 4.66358 14.6412 4.66358 14.74C4.66358 14.8388 4.6438 14.9366 4.6054 15.0276C4.567 15.1186 4.51077 15.2011 4.44001 15.27C4.37111 15.3406 4.28862 15.3964 4.19753 15.4343C4.10644 15.4721 4.00863 15.491 3.91001 15.49Z" />
```
Comment (evanpurkhiser) [inline static/app/icons/iconSwitch.tsx:8]: What did we change about the switch icon?

Comment (doralchan) [reply static/app/icons/iconSwitch.tsx:8]: Made it ever so slightly shorter (so @vuluongj20 can use it rotated in a selector)

Comment (evanpurkhiser) [reply static/app/icons/iconSwitch.tsx:8]: 👍 

I have rotated it before on _admin

**Pull request merged**